### PR TITLE
Added documentation for CIS release 1.11

### PR DESCRIPTION
**Pull request by snatra**
- Updated "_RELEASE-NOTES.rst_" file with v1.11 release notes

- Updated "_README.rst_"
        Added new route annotation support for OpenShift routes "virtual-server.f5.com/waf" in Table "Supported Route Annotations"

### CODE REVIEW

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -5,17 +5,27 @@ next-release
...
 `````````````````````
-* Support Path Based Routing for Openshift reencrypt routes.
-* OpenShift 4.1, Kubernetes 1.14 support and other dependency package upgrade.
-* Update OpenShift 3.11 GUI with Route admitted status based on BIG-IP response.
+* TBA 
 
 Bug Fixes
 `````````
-* Controller configured to manage-routes now exist with readable help message in logs when router-vserver-addr is not configured.
+* TBA
```
Comment (ssurenr) [inline docs/RELEASE-NOTES.rst]: Please remove these entries. A single blank line should be kept under `next-release`

Comment (snatra) [reply docs/RELEASE-NOTES.rst]: Ok Thanks

**Comment by ssurenr** [comment]
@snatra Squash the commits to a single commit when submitting PRs.

File: docs/RELEASE-NOTES.rst (hunk 1 above)
Comment (trinaths) [reply docs/RELEASE-NOTES.rst]: The requested changes are addressed.

**Pull request closed without merging**